README.md

```diff
@@ -1,10 +1,10 @@
 Idris 2
 =======
 
-[![Build Status](https://travis-ci.org/idris-lang/Idris2.svg?branch=master)](https://travis-ci.org/idris-lang/Idris2)
 [![Documentation Status](https://readthedocs.org/projects/idris2/badge/?version=latest)](https://idris2.readthedocs.io/en/latest/?badge=latest)
 ![Windows CI](https://github.com/idris-lang/Idris2/workflows/Windows%20CI/badge.svg)
-![Ubuntu CI](https://github.com/idris-lang/Idris2/workflows/Ubuntu%20CI/badge.svg)
+![Ubuntu CI bootstrap scheme](https://github.com/wchresta/Idris2/workflows/Ubuntu%20CI%20bootstrap%20scheme/badge.svg)
```

Comment:
I happened to notice that these URLs don't point to the official Idris 2 repo. Is this correct?

Reply:
Yikes, this is absolutely not correct. Thanks, will fix!